moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -214,14 +214,15 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
   bool getInterfaceDescription(moveit_msgs::msg::PlannerInterfaceDescription& desc)
   {
     auto req = std::make_shared<moveit_msgs::srv::QueryPlannerInterfaces::Request>();
-    auto response = query_service_->async_send_request(req);
+    auto future_response = query_service_->async_send_request(req);
 
     // wait until future is done
-    if (rclcpp::spin_until_future_complete(pnode_, response) == rclcpp::FutureReturnCode::SUCCESS)
+    if (rclcpp::spin_until_future_complete(pnode_, future_response) == rclcpp::FutureReturnCode::SUCCESS)
     {
-      if (!response.get()->planner_interfaces.empty())
+      const auto& response = future_response.get();
```

Comment:
Why adding reference here .? get returns a moved object, relying on object lifetime extension may introduce bugs in the future